LispLanguageGuide.md

```diff
@@ -0,0 +1,236 @@
+# GLaDOS - Lisp language usage guide
+## Table of contents
+- Basic syntax
+- Default symbols
```

Comment:
Please add a link system so that when you click here you to teleported to the right section